Example/Example ObjC/ScreenAPIViewController.m

```diff
@@ -48,134 +45,48 @@ - (IBAction)easyLaunchGiniVision:(id)sender {
     // 1. Create a custom configuration object
     GiniConfiguration *giniConfiguration = [GiniConfiguration new];
     giniConfiguration.debugModeOn = YES;
-    giniConfiguration.navigationBarItemTintColor = [UIColor whiteColor];    
+    giniConfiguration.navigationBarItemTintColor = [UIColor whiteColor];
+    giniConfiguration.multipageEnabled = YES;
+    giniConfiguration.fileImportSupportedTypes = GiniVisionImportFileTypesPdf_and_images;
+    giniConfiguration.openWithEnabled = YES;
+    giniConfiguration.qrCodeScanningEnabled = YES;
     
-    // 2. Create the Gini Vision Library view controller, set a delegate object and pass in the configuration object
-    UIViewController *vc = [GiniVision viewControllerWithDelegate:self withConfiguration:giniConfiguration importedDocument: NULL];
+    NSDictionary<NSString*, NSString*> *credentials = [[[CredentialsManager alloc] init] getCredentials];
     
+    GiniClient *client = [[GiniClient alloc] initWithClientId:credentials[@"client_id"]
+                                                 clientSecret:credentials[@"client_password"]
+                                            clientEmailDomain:credentials[@"client_domain"]];
```

Comment:
Didn't you make these key names constants for the Swift example? They look sad hardcoded like this.